nilearn/mass_univariate/_utils.py

```diff
@@ -89,6 +78,7 @@ def calculate_tfce(
             # is incrementally larger
             temp_arr3d = arr3d * sign
 
+            score_threshs = _return_score_threshs(arr3d, dh, two_sided_test)
```

Comment:
Let's move the setting of `score_threshs` to just before the start of the `for sign in signs:` loop.  There's no need to call it twice, and potentially a bug could creep in so that the two calls are given different arguments and produce different values.

Reply:
Agreed this can be hoisted out of the loop.